array-manipulation-3/src/unique.js

```diff
@@ -0,0 +1,27 @@
+/* exported unique */
+function unique(array) {
+  var obj = {};
+  var output = [];
+  for (var i = 0; i < array.length; i++) {
+    obj[array[i]] = array[i];
+  }
+  for (var x = 0; x < array.length; x++) {
+    for (var key in obj) {
+      if (obj[key] === array[x]) {
```

Comment:
You can remove the need for this `for...in` loop by just checking `if(obj[array[x]])`